src/main/java/com/service/DaoMailService.java

```diff
@@ -0,0 +1,34 @@
+package com.service;
+
+import com.dao.MailDao;
+import com.dto.MailDto;
+import com.model.Mail;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+@Service
+@AllArgsConstructor
+public class DaoMailService implements MailService {
```

Comment:
You should have a DAO level (responsible for persistence) and your service layer (responsible for business logic) separated. So, this class should be called `MailServiceImpl` or so